src/components/help/index.js

```diff
@@ -12,37 +13,37 @@ import styles from './styles.module.scss';
 export const Help = () => {
   return (
     <section className={styles.cards_wrapper}>
```

Comment:
There is a good practise, to mark tags `<article>` `<section> ` with `<h1>...<h6>`.
In `<h>` you should write description of the block and then use css property `display: none` if this `<h>` isn't needed in the design of the project